modules/aws/vendor-access/variables.tf

```diff
@@ -52,10 +52,17 @@ variable "eks_cluster_pattern" {
 }
 
 variable "external_id" {
+  default     = ""
```

Comment:
It should not have default value, users have to provide the Organization id as the external id